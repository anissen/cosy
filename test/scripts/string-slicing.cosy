{
    let s = 'hello'
    print s
    print s[1..4]
    print s[4..1]
}

{
    let s = 'hello'
    print s[1..]
    print s[2..]
    print s[..3]
    print s[..4]
    print s[..]
}