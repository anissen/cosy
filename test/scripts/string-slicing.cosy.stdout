hello
ell
lle
ello
llo
hel
hell
hello
